MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -1,72 +1,169 @@
+struct QuizStepViewModel {
+  let image: UIImage
+  let question: String
+  let questionNumber: String
+}
 
 
- Картинка: The Green Knight
- Настоящий рейтинг: 6,6
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: ДА
+// для состояния "Результат квиза"
+struct QuizResultsViewModel {
+  let title: String
+  let text: String
+  let buttonText: String
+}
 
 
- Картинка: Old
- Настоящий рейтинг: 5,8
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: НЕТ
+struct QuizQuestion {
+  let image: String
+  let text: String
+  let correctAnswer: Bool
+}
 
 
- Картинка: The Ice Age Adventures of Buck Wild
- Настоящий рейтинг: 4,3
- Вопрос: Рейтинг этого фильма больше чем 6?
- Ответ: НЕТ
+final class MovieQuizViewController: UIViewController {
+    
+    private let questions: [QuizQuestion] = [
+            QuizQuestion(
+                image: "The Godfather",
+                text: "Рейтинг этого фильма больше чем 5?",
+                correctAnswer: true),
+            QuizQuestion(
+                image: "The Dark Knight",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: true),
+            QuizQuestion(
+                image: "Kill Bill",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: true),
+            QuizQuestion(
+                image: "The Avengers",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: true),
+            QuizQuestion(
+                image: "Deadpool",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: true),
+            QuizQuestion(
+                image: "The Green Knight",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: true),
+            QuizQuestion(
+                image: "Old",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: false),
+            QuizQuestion(
+                image: "The Ice Age Adventures of Buck Wild",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: false),
+            QuizQuestion(
+                image: "Tesla",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: false),
+            QuizQuestion(
+                image: "Vivarium",
+                text: "Рейтинг этого фильма больше чем 6?",
+                correctAnswer: false)
+    ]
+    
+    private var currentQuestionIndex: Int = 0
+    private var correctAnswers: Int = 0
+
+    @IBOutlet private var counterLabel: UILabel!
+    @IBOutlet private var imageView: UIImageView!
+    @IBOutlet private var textLabel: UILabel!
+    
+
+    @IBAction private func noButtonClicked(_ sender: UIButton) {
+        let currentQuestion = questions[currentQuestionIndex]
+        let givenAnswer = false
+        self.showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
```

Comment:
Можно использовать короткий вариант без self
```suggestion
        showAnswerResult(isCorrect: givenAnswer == currentQuestion.correctAnswer)
```